src/librbd/exclusive_lock/PostAcquireRequest.cc

```diff
@@ -115,7 +115,7 @@ void PostAcquireRequest<I>::send_open_journal() {
   }
   if (!journal_enabled) {
     apply();
-    send_open_image_cache();
+    send_open_hook_points();
```

Comment:
Nit: same comment here re: function names throughout (e.g. `send_open_hook_points` -> `send_plugin_handle_acquire_exclusive_lock` or similar).

Reply:
Renamed send_open_hook_points() to send_process_plugin_acquire_lock(), and close_hook_points() to send_process_plugin_release_lock().